src/api/app/views/webui/request/beta_show_tabs/_review_summary.html.haml

```diff
@@ -0,0 +1,27 @@
+-# Don't render anything when project is nil
```

Comment:
```suggestion
-# FIXME: We don't render anything when the project got deleted because this nullifies the association. It should point to `Project.deleted_instance` or maybe the project got created with the same name again in the meanwhile?
```

Reply:
done